htmldocx/h2d.py

```diff
@@ -159,7 +192,7 @@ def delete_paragraph(paragraph):
 
 class HtmlToDocx(HTMLParser):
 
-    def __init__(self):
+    def __init__(self, ul_style=None, ol_style=None):
```

Comment:
Need to pass these to the instantiation of HtmlToDocx in handle_table

Reply:
This will be handled by the `copy_settings_from` method.